core/src/accounts_hash_verifier.rs

```diff
@@ -63,6 +63,10 @@ impl AccountsHashVerifier {
             .spawn(move || {
                 info!("AccountsHashVerifier has started");
                 let mut hashes = vec![];
+                // To support fastboot, we must ensure the storages used in the latest POST snapshot are
+                // not recycled nor removed early.  Hold an Arc of their AppendVecs to prevent them from
+                // expiring.
+                let mut last_snapshot_storages: Option<Vec<Arc<AccountStorageEntry>>> = None;
```

Comment:
Maybe we landed on this in a previous PR, but the type annotation here is not required.

Reply:
Ok, removed the type annotation